### ISSUE TITLE
mermaid diagram disappearing when toggling todo checkbox

### ISSUE THREAD

**Issue by pytzcarraldo** [notable/notable#621]
- **OS Version**: Ubuntu 18.04
- **Notable Version**: 1.4.0

### Current behavior

* Markdown note containing at least one GitHub-style checkbox `- [  ] todo` and a mermaid diagram.
* Toggling checkbox in render mode makes rendered mermaid diagram disappear.
* Mermaid diagram reappears when toggling `Ctrl + E`.  

### Expected behavior

* Checkbox toggling shouldn't affect mermaid diagram rendering.

**Comment by fabiospampinato** [comment]
I've noticed sometimes mermaid diagrams didn't get rendered too, but I wasn't able to reproduce this reliably, thanks for providing a way to do that 👍

**Comment by viktomas** [comment]
I'm able to reproduce just by switching to edit mode and back. This bug appeared either in `v1.5.0` or `v1.5.1`. A workaround is to open different note and then open the original note.

There are no errors logged to console in developer tools.

**Comment by ramtinsoltani** [comment]
@viktomas Same with me, it's happening on both of my Windows 10 x64 and Ubuntu 18.04 machines.

**Comment by shallinta** [comment]
Same as @viktomas  in `v1.5.1` on my macOS 10.14.4.

**Comment by Shagon94** [comment]
Same on the 1.5.1 version, can replicate;

```
#### mermaid

```mermaid
graph LR
  Install --> Tutorial[Read the tutorial]
  Tutorial --> Star
  Tutorial --> Share
  Tutorial -.-> mermaid[Realize how cool mermaid is]
```

all others work as expected, only the mermaid one is funky; I've uploaded how to reproduce here - 

https://gfycat.com/ifr/JampackedUnitedChinesecrocodilelizard

**Comment by fabiospampinato** [comment]
Fixed in v1.6.
